Grammar: "No results was found."

When I call .first() on a RealmResults object with no objects, it fires off the error message "No results was found." I found this out accidentally when my code bubbled it up to a Snackbar displayed to the user, and it threw me off. I know this is a very minor thing, but I suggest a grammatical correction of this error message. :) Thanks!

#### Goal

Get a suitable error message when there is no data to be returned

#### Expected Results

"No results were found."

#### Actual Results

"No results was found."

java.lang.IndexOutOfBoundsException: No results was found.
                                                                     at io.realm.RealmResults.first(RealmResults.java:221)
                                                                     at com.simplytapp.gane.data.user.datasource.LocalUserDataSource$2.call(LocalUserDataSource.java:69)
                                                                     at com.simplytapp.gane.data.user.datasource.LocalUserDataSource$2.call(LocalUserDataSource.java:65)
                                                                     at rx.internal.operators.OperatorMap$MapSubscriber.onNext(OperatorMap.java:66)
                                                                     at rx.internal.operators.OperatorFilter$FilterSubscriber.onNext(OperatorFilter.java:73)
                                                                     at io.realm.rx.RealmObservableFactory$5$1.onChange(RealmObservableFactory.java:143)
                                                                     at io.realm.rx.RealmObservableFactory$5$1.onChange(RealmObservableFactory.java:139)
                                                                     at io.realm.RealmResults.notifyChangeListeners(RealmResults.java:1032)
                                                                     at io.realm.RealmResults.notifyChangeListeners(RealmResults.java:1018)
                                                                     at io.realm.HandlerController.completedAsyncRealmResults(HandlerController.java:409)
                                                                     at io.realm.HandlerController.handleMessage(HandlerController.java:121)
                                                                     at android.os.Handler.dispatchMessage(Handler.java:98)
                                                                     at android.os.Looper.loop(Looper.java:148)
                                                                     at android.app.ActivityThread.main(ActivityThread.java:5422)
                                                                     at java.lang.reflect.Method.invoke(Native Method)
                                                                     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
                                                                     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
                                                                  Caused by: rx.exceptions.OnErrorThrowable$OnNextValue: OnError while emitting onNext value: io.realm.RealmResults.class
                                                                     at rx.internal.operators.OperatorMap$MapSubscriber.onNext(OperatorMap.java:70)
                                                                     at rx.internal.operators.OperatorFilter$FilterSubscriber.onNext(OperatorFilter.java:73) 
                                                                     at io.realm.rx.RealmObservableFactory$5$1.onChange(RealmObservableFactory.java:143) 
                                                                     at io.realm.rx.RealmObservableFactory$5$1.onChange(RealmObservableFactory.java:139) 
                                                                     at io.realm.RealmResults.notifyChangeListeners(RealmResults.java:1032) 
                                                                     at io.realm.RealmResults.notifyChangeListeners(RealmResults.java:1018) 
                                                                     at io.realm.HandlerController.completedAsyncRealmResults(HandlerController.java:409) 
                                                                     at io.realm.HandlerController.handleMessage(HandlerController.java:121) 
                                                                     at android.os.Handler.dispatchMessage(Handler.java:98) 
                                                                     at android.os.Looper.loop(Looper.java:148) 
                                                                     at android.app.ActivityThread.main(ActivityThread.java:5422) 
                                                                     at java.lang.reflect.Method.invoke(Native Method) 
                                                                     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
                                                                     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 

#### Steps & Code to Reproduce

Call .first() on a RealmResults object if there are no objects to be returned

#### Version of Realm and tooling
Realm version(s): 0.90.1

Android Studio version: 2.1.1

Which Android version and device: 6.0.1, Nexus 6P

--- a/realm/realm-library/src/main/java/io/realm/RealmResults.java
+++ b/realm/realm-library/src/main/java/io/realm/RealmResults.java
@@ -217,7 +217,7 @@ public final class RealmResults<E extends RealmModel> extends AbstractList<E> im
         if (size() > 0) {
             return get(0);
         } else {
-            throw new IndexOutOfBoundsException("No results was found.");
+            throw new IndexOutOfBoundsException("No results were found.");
         }
     }
 
@@ -230,7 +230,7 @@ public final class RealmResults<E extends RealmModel> extends AbstractList<E> im
         if (size > 0) {
             return get(size - 1);
         } else {
-            throw new IndexOutOfBoundsException("No results was found.");
+            throw new IndexOutOfBoundsException("No results were found.");
         }
     }
 

